proto/storage/notification.proto

```diff
@@ -23,16 +23,17 @@ message Notification {
     string resource_type                    = 7 [(gogoproto.moretags) = 'search:"Resource,hidden"'];
     string resource_id                      = 8;
     int64 occurrences                       = 9;
-    google.protobuf.Timestamp last_occurred = 10;
+    google.protobuf.Timestamp last_occurred = 10 [(gogoproto.moretags) = 'search:"Last Updated,hidden"'];
     google.protobuf.Timestamp created_at    = 11 [(gogoproto.moretags) = 'search:"Created Time,hidden"'];
 }
 
 // Notification type is the storage representation of the Notification type.
 //
 // Refer to v1.NotificationType for a more detailed doc.
 enum NotificationType {
-    NOTIFICATION_TYPE_GENERIC     = 0;
-    NOTIFICATION_TYPE_LOG_MESSAGE = 1;
+    NOTIFICATION_TYPE_UNKNOWN     = 0;
```

Comment:
This does not feel backwards compatible as the underlying meaning of the value has changed.  This matters since we no longer copy databases around.

Reply:
OR is all this stuff new-ish and not really in use yet?  Which would make my comment meaningless.